docs/source/daml-integration-kit/index.rst

```diff
@@ -101,18 +101,34 @@ support for running DAML.
 
    * - Ledger
      - Status
-     - Developer
+     - Integration Support
```

Comment:
I don't quite understand why this changed?

Reply:
@bethaitman -- from the PR it is mentioned the Integration company that would run the network upon which a solution would be hosted, which is slightly different to Developer.  Not sure if there are actually two things to show here -- who performed the initial development and who will support integration in a running environment.  Probably needs a Chris/Simon/Shaul conversation - I'll also bring up in standup today.

Reply:
Cool, thanks for the clarification. I don't mind which you go for (or if you want to list both the developer and who would host the integration) just as long as the information is clear. I wonder if something like "Integration hosted by" or "Ledger hosted by" might be a better name for the latter.